cobbler.spec

```diff
@@ -340,6 +358,7 @@ sed -i -e "s/SECRET_KEY = ''/SECRET_KEY = \'$RAND_SECRET\'/" %{_datadir}/cobbler
 %{python3_sitelib}/cobbler/
 %{python3_sitelib}/cobbler-*
 %{_unitdir}/cobblerd.service
+%{_sbindir}/rccobblerd
```

Comment:
This will _only_ work on SUSE distributions, so this needs to be conditioned out.

Reply:
Done

Reply:
I am not used yet to reviews on GH (we use RBCommons instead), so bear with me.
Since I can't select older lines, I write it here.
The RAND_SECRET is not working in the `sed` that follows, WHEN it contains an ampersand (`&`). This happens 98 out of 100 times. See #2238 
Ah. just add a diff instead for the other issues mentioned there:

```diff
# diff cobbler.spec-mine cobbler.spec
39c39
< %define apache_webconfigdir /etc/apache2/conf-available
---
> %define apache_webconfigdir /etc/apache2/conf.d
262,269c262
< mkdir -p %{buildroot}%{tftpboot_dir}/boot
< mkdir %{buildroot}%{tftpboot_dir}/etc
< mkdir %{buildroot}%{tftpboot_dir}/grub
< mkdir %{buildroot}%{tftpboot_dir}/images
< mkdir %{buildroot}%{tftpboot_dir}/images2
< mkdir %{buildroot}%{tftpboot_dir}/ppc
< mkdir %{buildroot}%{tftpboot_dir}/pxelinux.cfg
< mkdir %{buildroot}%{tftpboot_dir}/s390x
---
> mkdir -p %{buildroot}%{tftpboot_dir}/{boot,etc,grub,images{,2},ppc,pxelinux.cfg,s390x}
326,327c319
< #RAND_SECRET=$(openssl rand -base64 40 | sed 's/\//\\\//g')
< RAND_SECRET=$(head /dev/urandom | tr -dc 'A-Za-z0-9!"#$%'\''()*+,-./:;<=>?@[\]^_`{|}~' | head -c 57 ; echo '')
---
> RAND_SECRET=$(openssl rand -base64 40 | sed 's/\//\\\//g')
```